drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/InsertColumnCommand.java

```diff
@@ -40,7 +40,7 @@
     private String columnId;
     private int columnIndex;
     protected boolean isRight;
-    private boolean asProperty;
+    protected boolean asProperty;
```

Comment:
Sorry @gitgabrio. I mean this change from private to protected. If you use split the test and use the constructor this does not have to change and the class works as it is expected.

Reply:
@Rikkola 
Done.